cli.js

```diff
@@ -1149,7 +1170,7 @@ function status(source, json, argv) {
 
               return Promise.all(
                 eventMap.map(eventName => {
-                  let re = new RegExp(moduleName + '\\.' + eventName);
+                  let re = new RegExp('(' + moduleName + '\\.' + eventName + '|\\[' + moduleName + " " + eventName +')');
```

Comment:
This one is to support ObjC syntax, where method calls are [Avo methodName(OptionalParams)] and we look for `[Avo methodName`